tests/tape/interfaces/test_qnode_jax.py

```diff
@@ -17,7 +17,8 @@
 jnp = pytest.importorskip("jax.numpy")
 import numpy as np
 import pennylane as qml
-from pennylane.tape import JacobianTape, qnode, QNode, QubitParamShiftTape

```

Comment:
Should this file be moved to `/tests/interfaces/test_qnode_jax.py`? Same with `test_tape_jax.py` ?

Reply:
Very good catch!